### ISSUE TITLE
Broken generation for 0.5

### ISSUE THREAD

**Issue by mrspacemankey** [coveooss/json-schema-for-humans#15]
Hi @dblanchette ,

I have run into the following issue with the new 0.5 version. To reproduce it I started with a fresh environment:

```bash
python -m venv env
source env/bin/activate
pip install json-schema-for-humans==0.5
generate-schema-doc document.json document.html
```

I get the following error:

```
Traceback (most recent call last):
  File "***/env/bin/generate-schema-doc", line 8, in <module>
    sys.exit(main())
  File "***/env/lib64/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "***/env/lib64/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "***/env/lib64/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "***/env/lib64/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "***/env/lib64/python3.6/site-packages/json_schema_for_humans/generate.py", line 400, in main
    schema_file, result_file, minify, deprecated_from_description, default_from_description, expand_buttons
  File "***/env/lib64/python3.6/site-packages/json_schema_for_humans/generate.py", line 361, in generate_from_file_object
    expand_buttons,
  File "***/env/lib64/python3.6/site-packages/json_schema_for_humans/generate.py", line 311, in generate_from_schema
    rendered = template.render(schema=schema, schema_path=schema_path, expand_buttons=expand_buttons)
  File "***/env/lib64/python3.6/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "***/env/lib64/python3.6/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "***/env/lib64/python3.6/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 224, in top-level template code
  File "***/env/lib64/python3.6/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 138, in template
  File "***/env/lib64/python3.6/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 69, in template
  File "***/env/lib64/python3.6/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 156, in template
  File "***/env/lib64/python3.6/site-packages/json_schema_for_humans/generate.py", line 93, in resolve_ref
    target_path.append(ref_path_segment)
```

I checked the same with using version 0.4.1 and that still works (as before).

**Comment by dblanchette** [comment]
I was able to reproduce the error using the schema at https://json-schema.org/draft/2019-09/schema. There is a ref that does not start with the pound sign `#`. Can you confirm it is also the case with the schema you are using?

The fix is simple enough, but it means references that does not start with the pound sign won't be retrieved.

**Comment by dblanchette** [comment]
The problem was simpler than I initially thought. I just forgot to pass down the schema_path for the case where the schema is not an object.

Fix released to pypi in version 0.5.1
